webpack.config.production.js

```diff
@@ -1,13 +1,55 @@
 const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const ImageminPlugin = require('imagemin-webpack-plugin').default
 
 module.exports = {
   output: {
     filename: 'javascripts/[name].[contenthash:8].js',
   },
 
+  module: {
+    rules: [
+      {
+        test: /\.(scss|css)$/,
+        use: [
+          {
+            loader: MiniCssExtractPlugin.loader,
+            options: {
+              hmr: false,
+            },
+          },
+        ],
+      },
+      {
+        test: /\.(woff(2)?|ttf|eot|svg)(\?v=\d+\.\d+\.\d+)?$/,
+        use: [
+          {
+            loader: 'file-loader',
```

Comment:
You might not need to define this line as it may be taken from the shared config, maybe try it without?

Reply:
`merge.smart` needs the `loader` name to match and merge this with the shared config